app/lib/hosting_environment.rb

```diff
@@ -21,9 +21,13 @@ def self.hostname
 
   def self.phase
     if production?
-      'beta'
+      'Beta'
+    elsif sandbox_mode?
+      'Sandbox'
+    elsif qa?
+      'QA'
```

Comment:
You can avoid this conditional by adding `QA` to config/initializers/inflector.rb.

Reply:
Oooh, nice!